recipes/recipes_emscripten/xxhash/recipe.yaml

```diff
@@ -0,0 +1,41 @@
+context:
+  name: xxhash
+  version: 0.8.2
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/Cyan4973/${{ name }}/archive/v${{ version }}.tar.gz
+  sha256: baee0c6afd4f03165de7a4e67988d16f0f2b257b51d0e3cb91909302a26a79c4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - make  # [unix]
```

Comment:
the selectors like ` # [unix]` are not working with the rattler-recipe format. Probably many recipes still have these sectors in them, but any of these are superfluous